cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/AbstractTicketDelegator.java

```diff
@@ -73,7 +75,7 @@ public final TicketGrantingTicket getGrantingTicket() {
     }
 
     @Override
-    public final long getCreationTime() {
+    public final ZonedDateTime getCreationTime() {
```

Comment:
which we currently use, so this should be good with JPA. I haven't run any local tests with it yet to confirm.